src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { useState } from "react";
+
+import SearchHeader from "../SearchHeader/SearchHeader";
+import PokemonData from "../PokemonData/PokemonData";
+import Footer from "../Footer/Footer";
+import { fetchPokemon } from "../../utils/FetchPokemon";
+
+function InfoPage() {
+  const [pokeInfo, setPokeInfo] = useState();
+  const [loading, setLoading] = useState(false);
+
+  const getPokemon = async (query) => {
+    try {
+      setLoading(true);
+      const response = await fetchPokemon(query);
+      const results = await response.json();
```

Comment:
`      const results = await response.json();
`

You need to check every response from the server  to invoke an error if the response is not `ok`.


```js
      if (!response.ok) {
        throw new Error(`Error ${response.status}`);
      }
      const results = await response.json();

```